src/containers/org/OrgContainer.js

```diff
@@ -196,59 +178,23 @@ const OrgContainer = ({
               )
             }
           </StackGrid>
-          <Typography variant="h2">Data Sets</Typography>
-          <StackGrid gap={8}>
-            <SearchForm
-                onSubmit={(query :string) => dispatchDataSetSearch({ query })}
-                searchQuery={searchQuery}
-                searchRequestState={searchOrgDataSetsRS} />
-            <GapGrid gap={8}>
-              <Typography variant="subtitle2">Search Options</Typography>
-              <Flip flip={isOpenSearchOptions}>
-                <IconButton aria-label="toggle search options" onClick={toggleSearchOptions}>
-                  <FontAwesomeIcon fixedWidth icon={faChevronDown} size="xs" />
-                </IconButton>
-              </Flip>
-            </GapGrid>
-            <Collapse in={isOpenSearchOptions}>
-              <Box maxWidth={240}>
-                <Typography gutterBottom variant="subtitle1">Flags</Typography>
-                <Select
-                    isClearable
-                    onChange={(option :?ReactSelectOption<string>) => setFlag(option?.value)}
-                    options={ES_FLAG_TYPE_RS_OPTIONS} />
-              </Box>
-            </Collapse>
-          </StackGrid>
-          {
-            !isStandby(searchOrgDataSetsRS) && (
-              <PaginationToolbar
-                  count={searchTotalHits}
-                  onPageChange={({ page, start }) => dispatchDataSetSearch({ page, start })}
-                  page={searchPage}
-                  rowsPerPage={MAX_HITS_10} />
-            )
-          }
-          {
-            isPending(searchOrgDataSetsRS) && (
-              <Spinner />
-            )
-          }
-          {
-            isSuccess(searchOrgDataSetsRS) && searchHits.isEmpty() && (
-              <Typography align="center">No data sets.</Typography>
-            )
-          }
-          {
-            isSuccess(searchOrgDataSetsRS) && !searchHits.isEmpty() && (
-              searchHits.valueSeq().map((searchHit :Map) => (
-                <DataSetSearchResultCard
-                    dataSet={searchHit}
-                    key={searchHit.get('id')}
-                    organizationId={organizationId} />
-              ))
-            )
-          }
+          <Tabs
+              aria-label="tabs"
+              indicatorColor="primary"
+              onChange={handleChangeTab}
+              textColor="primary"
+              value={tab}>
+            <Tab label="Data Sets" value="datasets" />
+            <Tab label={collabTabText} value="collaborations" />
+          </Tabs>
+          <StyledTabPanel show={tab === 'datasets'}>
+            <OrgDataSetsContainer organizationId={organizationId} />
+          </StyledTabPanel>
+          <StyledTabPanel show={tab === 'collaborations'}>
+            <CollaborationsParticipationContainer
+                collaborations={collaborationsByOrganizationId}
+                type="organization" />
+          </StyledTabPanel>
```

Comment:
Both `OrgDataSetsContainer` and `CollaborationsParticipationContainer` are mounted at all times. Is this intentional?

![Screen Shot 2021-07-19 at 4 02 26 PM](https://user-images.githubusercontent.com/440299/126238434-fa7cbfa9-7e88-430a-894d-07000779597e.png)

Reply:
Yes.